site/docs/foundations/data-visualization/chart-anatomy.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Chart anatomy
+layout: DetailTechnical
+sidebar:
+  priority: 4
+---
+
+Understanding the principles of chart anatomy is important to learning how the different parts of Salt charts work. 
+
+This will include: 
+
+- Axis title
+- Axis label
+- Legend
+- Grid lines
+
+<Image
+  src="/img/foundations/dataviz-anatomy.png"
+  srcDark="/img/foundations/dataviz-anatomy-dark.png"
+  alt="Diagram of a typical bar chart anatomy."
+/>
+
+## Titles
+
+The chart’s title should summarize the data on display. If revealing key findings, supporting an argument, story or message, capture it in the title.
+
+## Axes
```

Comment:
Would you benefit from an image in this section, that puts your recommendation into practice?